### PR TITLE
Raise exceptions on user or database if a resource with the same name is found.

### PR DESCRIPTION
**Pull request by TimSimpsonR**
- Updated reference guest and guest API.

### CODE REVIEW

File: reddwarf/extensions/mysql/models.py [hunk 1]
```diff
@@ -99,7 +99,15 @@ def __init__(self, name, password, databases):
     def create(cls, context, instance_id, users):
         # Load InstanceServiceStatus to verify if it's running
         load_and_verify(context, instance_id)
-        create_guest_client(context, instance_id).create_user(users)
+        client = create_guest_client(context, instance_id)
+        for user in users:
```
Comment (cp16net) [inline reddwarf/extensions/mysql/models.py:103]: can you add a small comment of what this block is doing for easy reference?

Comment (TimSimpsonR) [inline reddwarf/extensions/mysql/models.py:103]: I don't know, I feel like what the code is doing and why is made clear by the exception name of "UserAlreadyExists". Even if someone doesn't understand how it works right off they can at least see its checking if the user is already there by scanning the code.

Comment (cp16net) [inline reddwarf/extensions/mysql/models.py:103]: alright just a suggestion.

**Comment by cp16net** [comment]
+1

**Comment by rnirmal** [comment]
+1... but thinking of it, would have been simpler to just add a call to check if the user/database exists. Didn't think of that yesterday.

Good work on getting this fixed up asap

**Comment by TimSimpsonR** [comment]
@rnirmal   In retrospect I think you're right. It would've been a bit more work in the guests but the real bugs ended up lurking in the checks in RDL.

**Comment by hub-cap** [comment]
Supa fast!!!

**Pull request merged**